en/docs/install-and-setup/setup/security/enabling-hostname-verification.md

```diff
@@ -16,6 +16,18 @@ The hostname verification is **disabled** by default. This is done using the `o
     -Dhttpclient.hostnameVerifier=AllowAll \
 ```
 
+When applying the strict hostname verification, you need to change the configuration in the `site/public/conf/settings.js` file of the Publisher and Admin Portal and the `site/public/theme/setting.js` file of the Developer Portal.
```

Comment:
Let's add this as a note and highlight strict hostname verification. Hope that would be fine.

Reply:
Fixed.